<launch>
    <node pkg="artificial_objects" type="artificial_object"
          name="artificial_object1" output="screen">

    <param name="name_space" type="string" value="human" />
    <param name="frame_id" type="string" value="map" />
    <rosparam param="color"> [1,0,0]</rosparam>
    <rosparam param="freq"> 1</rosparam>
    <rosparam param="mark_size"> 0.05 </rosparam>
      <rosparam param="x">[4.20469456, 4.20469456, 4.20469456, 4.1538932 , 3.99138658,
       3.69611307, 3.46173338, 3.30489134, 3.01177414, 2.91448764,
       2.56961841, 2.52293481, 2.34509332, 2.14038449, 1.90391563]</rosparam>
    <rosparam param="y">[1.59774804 , 1.59774804 , 1.59774804, 1.72573372, 1.83545505,
       1.78209337, 1.92104339, 2.01183193, 2.04859039, 2.07748556,
       2.1029878 , 2.18356163, 2.00349128, 1.86427669, 1.95638399]</rosparam>
    </node>

    <node pkg="artificial_objects" type="artificial_object"
          name="artificial_object2" output="screen">

    <param name="name_space" type="string" value="robot" />
    <param name="frame_id" type="string" value="map" />
    <rosparam param="color"> [0.5,0.3,1]</rosparam>
    <rosparam param="freq"> 3</rosparam>
    <rosparam param="mark_size"> 0.05 </rosparam>
    <rosparam param="x">  [3.79425988,3.79425988,3.79425988,3.79425988,3.79425988,3.79425988,3.79425988,3.79425988,3.79425988,3.79425988,3.79425988,  3.77859847,  3.74942437,  3.76761666,  3.73003434,
        3.68394526,  3.66132553,  3.62292711,  3.60807746,  3.58337806,
        3.54156245,  3.55751375,  3.51224089,  3.41711094,  3.41387701,
        3.40904232,  3.33773568,  3.32066335,  3.2861225 ,  3.2332004 ,
        3.1687611 ,  3.14702037,  3.08064404,  3.02351596,  3.036244  ,
        2.97821886,  2.90045342,  2.88365049,  2.77946406,  2.76582673,
        2.76160412,  2.72710602,  2.67805121,  2.57811214,  2.50180802,
        2.46203951,  2.44090526,  2.34010764,  2.32021515,  2.32067965,
        2.23751443,  2.11536645,  2.10945843,  2.04836371,  1.99733923,
        1.92209594,  1.90133104,  1.81919128,  1.74235917,  1.69934856,
        1.62412418,  1.58720013,  1.52820005,  1.47444696,  1.43313284,
        1.39602626,  1.26384192,  1.2054227 ,  1.16892296,  1.12160007,
        1.08380235,  0.98377601,  0.87308174,  0.86948393,  0.76945709,
        0.72069518,  0.61589107,  0.64126993,  0.50328919,  0.54221497]</rosparam>
    <rosparam param="y">[4.27426954,4.27426954,4.27426954,4.27426954,4.27426954,4.27426954,4.27426954,4.27426954,4.27426954,4.27426954,4.27426954, 4.26204301, 4.27759767, 4.19055081, 4.13746406,
       4.09983525, 4.02760853, 4.0300805 , 3.98625314, 3.95836674,
       3.87270546, 3.84977075, 3.78741085, 3.78178815, 3.76076639,
       3.72851438, 3.59451568, 3.64621566, 3.53236775, 3.53035293,
       3.52547515, 3.44212444, 3.37718492, 3.3831605 , 3.3362438 ,
       3.30765205, 3.2426098 , 3.22782483, 3.19812222, 3.14744748,
       3.10221819, 3.04440501, 3.03305816, 2.99856091, 2.91385695,
       2.89968779, 2.84426055, 2.77537281, 2.79245611, 2.73669875,
       2.69919764, 2.62667856, 2.64172891, 2.54926137, 2.49375847,
       2.47781458, 2.46602455, 2.37880909, 2.4551938 , 2.29516416,
       2.26435351, 2.25535507, 2.22197042, 2.13213454, 2.11747904,
       2.1057669 , 2.04813159, 1.95085854, 1.93839519, 1.92750006,
       1.8941504 , 1.8716096 , 1.79329456, 1.77994292, 1.70108479,
       1.69182516, 1.64570522, 1.60646252, 1.55618483, 1.6131738]</rosparam>
    </node>

</launch>